### PR TITLE
Multimeter python bridge script imporvement

### LINKED ISSUE #329: Improve logic on python script for Multimeter
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request

* **What is the current status?**
- The script is not suitable for measuring very accurate readings

* **What is the expected behaviour?**
- The output being spit out by the script should take care of precision and should display the values accordingly

* **Please tell us about your environment:**
  - OS with version number: Ubuntu 18.04

* **Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #329 

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bug fix
- [ ] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**
- Added a preprocessor to provide a formatted output for readings
- Depending on the range of output, a prefix will be added "M", "K", "m", "μ" in the output. 
- Data is now rounded off to the third decimal place

* **Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: scripts/multimeter.py [hunk 1]
```diff
@@ -27,6 +27,18 @@ def get_config(self):
         print(json.dumps(output))
         sys.stdout.flush()
 
+    def precision_control(self, value):
+        if value >= 1e6:
+            return (round(value / 1e6, 3), 'M')
+        if value >= 1e3:
+            return (round(value / 1e3, 3), 'K')
```
Comment (CloudyPadmal) [inline scripts/multimeter.py]: Not simple k? :thinking:

Comment (AakashMallik) [reply scripts/multimeter.py]: sorry I don't follow. Simple K? :sweat_smile:

Comment (CloudyPadmal) [reply scripts/multimeter.py]: Yes SI Standards. Simple k for kilo; capital K for Kelvin

Comment (AakashMallik) [reply scripts/multimeter.py]: oh.. you meant lower case... got it.. my bad. Will fix it ASAP.

**Comment by AakashMallik** [comment]
@CloudyPadmal I have changed it to lower case. Being an engineer, making such mistakes makes me feel embarrassed... :joy: Thank you for your detailed review.

**Pull request merged**